### ISSUE TITLE
addResource scrollTo param broken

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar-scheduler#335]
addResource with the scrollTo=true param doesn't work

```js
$('#calendar').fullCalendar(
	'addResource',
	{ title: title },
	true // scroll to the new resource?
);
```

https://fullcalendar.io/js/fullcalendar-scheduler-1.6.2/demos/dynamic-add-remove.html

**Comment by arshaw** [comment]
queued up for release

**Comment by arshaw** [comment]
Released in [v1.8.0](https://github.com/fullcalendar/fullcalendar-scheduler/releases/tag/v1.8.0)
